### PR TITLE
Making the encoding consistent- changing lowercase utf to uppercase

### PR DESCRIPTION
**Pull request by charvimehradu**
Pull Request for Issue #39927 .

### Summary of Changes

Replacing lowecase utf with uppercase.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by sandewt** [comment]
Related to  #26221

**Comment by richard67** [comment]
Drone and appveyor are failing due to syntax errors

File: .editorconfig [hunk 1]
```diff
@@ -7,7 +7,7 @@ root = true
 [*]
 indent_style = tab
 end_of_line = lf
-charset = utf-8
+charset = UTF-8
```
Comment (sandewt) [inline .editorconfig]: I see you changed the code everywhere. Perhaps it is wiser to initially focus only on the manifest files, the `.xml` files.

**Comment by bembelimen** [comment]
Hello @charvimehradu ,
Thanks for this PR, it's a very good start, now we just have to fix this minor stuff to get it merge-ready.

**Comment by brianteeman** [comment]
You can find details of the minor changes that are needed here https://ci.joomla.org/joomla/joomla-cms/62238/1/8

File: .editorconfig (hunk 1 above)
Comment (joomdonation) [reply .editorconfig]: Agree with @sandewt . I think for this PR, you should make change for xml files only. For example, I also the code changed to below:

```php
header('Content-Type: text/plain; charset=UTF-8');
```
However, from what I see from various source, for example [https://www.w3.org/International/articles/http-charset/index](https://www.w3.org/International/articles/http-charset/index) , it should be

```php
header('Content-Type: text/plain; charset=utf-8');
```
Your change might still work, but to be safe, I would start with xml files only.

**Comment by richard67** [comment]
@charvimehradu Please check also the review comments by @sandewt and @joomdonation and revert all changes which are not in XML files. Thanks in advance.

**Comment by richard67** [comment]
@charvimehradu Your commit to revert the changes (2nd last commit) had merge conflicts. They are marked with `>>>>` and `====` and `<<<<` in the files. You need to resolve these conflicts (just search for the mentioned patterns and you will find them. Let us know if you need help with that. Alkternatively, and maybe that would be easier for you, just make a new pull request and then close this one in favour of the new one. Thanks in advance, and thanks for your work so far.

**Comment by charvimehradu** [comment]
yes thankyou @richard67 I'll just create a new PR for this one and then close this.

File: .editorconfig (hunk 1 above)
Comment (sandewt) [reply .editorconfig]: > However, from what I see from various source, 

Here's another example, quite confusing:

https://developer.mozilla.org/en-US/docs/Web/XML/XML_introduction

@charvimehradu  First it must be clear whether we use lowercase or uppercase letters

**Comment by charvimehradu** [comment]
Thankyou everyone for your feedback and suggestions on my pull request. I have addressed all the suggested changes, and have created a new pull request #39951 . I'm optimistic that this time it will pass all the checks. @sandewt @richard67 @bembelimen @brianteeman @joomdonation

**Pull request closed without merging**